### PR TITLE
Blog Feature

### PR DESCRIPTION
**Pull request by Cscotttaakan**
## Description
Blog feature, CRUD

## Pls do 
- [x] Tested and working
- [x] Unit tests

## Test plan
'make test' and check on postman
Create
  Create Blog Post Postman Screenshot

Retrieve
  Create Blog Post By ID Postman Screenshot

Search  Query Blog Post Postman Screenshot

### CODE REVIEW

File: src/blog/models.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+class BlogPost(models.Model):
+    """
+    Model containing title, body, and images
+    """
+    title = models.CharField(max_length=250)
+    author = models.CharField(max_length=60)
+    user = models.ForeignKey(User, related_name='user',
+            on_delete=models.CASCADE, null=True, blank=True)
+    body = models.TextField()
+    publish = models.DateTimeField(default=timezone.now)
```
Comment (DavidWChen) [inline src/blog/models.py]: Make then changes we mention: published as a bool & edit search to reflect this

File: src/blog/models.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    class Meta:
+        ordering = ('-filename',)
+
+    def __str__(self):
+        return self.filename
+
+    #delete S3 reference
+    def delete(self):
+        #pictureid = str(self.picture).split('/')[1]
```
Comment (DavidWChen) [inline src/blog/models.py]: Comments in code are great; commented out code, not as great

File: src/blog/views.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            for key in request.FILES:
+                picture = BlogPicture.objects.create(
+                            filename = request.FILES[key].name,
+                            blog = blog,
+                            picture = request.FILES[key],
+                        )
+                picture.save()
+            blog.save()
+
+            #Cycles through keys in files for multiple image upload
```
Comment (DavidWChen) [inline src/blog/views.py]: Same here, why is there commented out code?

**Comment by Cscotttaakan** [comment]
#99

**Pull request merged**